_posts/2015-11-23-my-first-couple-of-months-at-codurance.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+name: my-first-couple-of-months-at-codurance
+title: 'My first couple of months at Codurance'
+date: 2015-11-23 09:00:00 +00:00
+author: Mani Sarkar
+image:
+    src: "/assets/img/custom/blog/Five\ Characteristics\ of\ a\ Great\ Company\ Culture.jpg"
+    attribution:
+        text: Together we are a formidable force
+tags:
+- craftsmanship
+- apprenticeship
+- community
+- mentor
+- mentee
+- mentoring
+---
+
+### Some background
+
+Some of you may know me from the various meetups in the city, especially my attendance at a number of [LJC](www.meetup.com/Londonjavacommunity/) and [LSCC](www.meetup.com/london-software-craftsmanship) meetup events. Attending these events I learnt about various conferences like **Devoxx**, **SoCraTes**, **JAX London**, **Java2Days**, **OpenFest**, and some others, and have had the good fortune of being at such events and even presenting at some of them. My subject was usually around [Adopt OpenJDK](https://java.net/projects/adoptopenjdk/pages/AdoptOpenJDK) whilst wearing my [LJC](www.meetup.com/Londonjavacommunity/) Advocate hat, but I also took the chance to present on other topics.
+
```

Comment:
From "Attending these events... " until the end of the paragraph. It's a bit too verbose and bumpy. You could replace it with something like "At these community events, I learned about some interesting conferences like Devoxx..., which I ended up attending and later presenting on various topics including the Adopt OpenJDK."

Remove the thing about wearing a LJC advocate hat.